.github/workflows/ci_benchmark.yml

```diff
@@ -0,0 +1,99 @@
+### Copied from https://github.com/scikit-image/scikit-image/blob/main/.github/workflows/benchmarks.yml
+
+name: Benchmark
+
+on:
+  pull_request:
+    types: [labeled]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  benchmark:
+    if: contains(github.event.label.name, 'benchmark') || github.event_name == 'workflow_dispatch'
+    name: Linux
```

Comment:
This name doesn't really tell me anything useful, so either make it less cryptic or just remove it altogether?

```suggestion
```